src/App.js

```diff
@@ -1,23 +1,17 @@
 import React from "react";
```

Comment:
Importa los hooks que vas a utilizar, así el componente queda más reducido

```suggestion
import React, { useState } from "react";
```